scene/realistic-lighting-and-shadows/src/main/java/com/esri/samples/realistic_lighting_and_shadows/RealisticLightingAndShadowsController.java

```diff
@@ -0,0 +1,157 @@
+import com.esri.arcgisruntime.mapping.ArcGISScene;
+import com.esri.arcgisruntime.mapping.ArcGISTiledElevationSource;
+import com.esri.arcgisruntime.mapping.Basemap;
+import com.esri.arcgisruntime.mapping.Surface;
+import com.esri.arcgisruntime.mapping.view.AtmosphereEffect;
+import com.esri.arcgisruntime.mapping.view.Camera;
+import com.esri.arcgisruntime.mapping.view.LightingMode;
+import com.esri.arcgisruntime.mapping.view.SceneView;
+
+public class RealisticLightingAndShadowsController {
+
+  @FXML private SceneView sceneView;
+  @FXML private Label timeLabel;
+  @FXML private Slider timeSlider;
+  @FXML private ComboBox<LightingMode> comboBox;
+  private Surface surface;
+  private Calendar calendar;
+  private SimpleDateFormat dateFormat;
+
+  public void initialize() {
+    try {
+
+      // create a scene and add a basemap to it
+      ArcGISScene scene = new ArcGISScene();
+      scene.setBasemap(Basemap.createTopographic());
+
+      // add the scene to the scene view
+      sceneView.setArcGISScene(scene);
+
+      // add a base surface for elevation data to the scene
+      surface = new Surface();
+      ArcGISTiledElevationSource elevationSource = new ArcGISTiledElevationSource("http://elevation3d.arcgis.com/arcgis/rest/services/WorldElevation3D/Terrain3D/ImageServer");
+      surface.getElevationSources().add(elevationSource);
+      scene.setBaseSurface(surface);
+
+      // add a scene layer
+      ArcGISSceneLayer sceneLayer = new ArcGISSceneLayer("http://tiles.arcgis.com/tiles/P3ePLMYs2RVChkJx/arcgis/rest/services/DevA_BuildingShells/SceneServer/layers/0");
+      scene.getOperationalLayers().add(sceneLayer);
+
+      // add a camera and initial camera position
+      Camera camera = new Camera(45.54605153789073, -122.69033380511073, 941.0002111233771, 162.58544227544266, 60.0,0.0);
+      sceneView.setViewpointCamera(camera);
+
+      // set atmosphere effect to realistic
+      sceneView.setAtmosphereEffect(AtmosphereEffect.REALISTIC);
+
+      // set a calendar with a date and time
+      calendar = new GregorianCalendar(2018, 7, 10, 12, 00, 0);
+      calendar.setTimeZone(TimeZone.getTimeZone("PST"));
+
+      // set the sun time to the calendar
+      sceneView.setSunTime(calendar);
+
+      // format the string to just return the date and time (hours and minutes)
+      dateFormat = new SimpleDateFormat("EEE MMM dd HH:mm");
+      dateFormat.setTimeZone(TimeZone.getTimeZone("PST"));
+      String formattedDateAndTime = dateFormat.format(calendar.getTime());
+
+      // set a label to display the formatted date and time
+      timeLabel.setText(formattedDateAndTime);
+
+      // set the slider to display tick labels as time strings
+      timeSlider.setLabelFormatter(new SliderStringConverter());
+
+      // add the lighting modes to the combo box
+      comboBox.getItems().add(LightingMode.NO_LIGHT);
+      comboBox.getItems().add(LightingMode.LIGHT);
+      comboBox.getItems().add(LightingMode.LIGHT_AND_SHADOWS);
```

Comment:
you can reduce the number of lines here by using `addAll`: 
```suggestion
      comboBox.getItems().addAll(LightingMode.LIGHT, LightingMode.LIGHT_AND_SHADOWS, LightingMode.NO_LIGHT);

```